adapters/adagio/adagiotest/exemplary/multiformat-web.json

```diff
@@ -0,0 +1,222 @@
+{
+  "mockBidRequest": {
+    "test": 1,
+    "id": "some-request-id",
+    "device": {
+      "ua": "test-user-agent",
+      "ip": "123.123.123.123",
+      "language": "en",
+      "dnt": 0
+    },
+    "tmax": 1000,
+    "user": {
+      "buyeruid": "awesome-user"
+    },
+    "site": {
+      "page": "test.com",
+      "publisher": {
+        "id": "123456789"
+      }
+    },
+    "imp": [
+      {
+        "id": "banner-impression-id",
+        "banner": {
+          "w":320,
+          "h":50
+        },
+        "ext": {
+          "bidder": {
+            "organizationId": "1000",
+            "site": "site-name",
+            "placement": "ban_atf"
+          }
+        }
+      },
+      {
```

Comment:
This looks like a multi-imp test rather than a multiformat test. Please consider renaming it accordingly and adding a multiformat test where a given imp has multiple formats specified for example, both banner and video.